src/ChartContainer/ShareDropDown.js

```diff
@@ -2,15 +2,25 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 import { makeStyles } from '@material-ui/styles';
-import { Button, Grid, Link, Typography } from '@material-ui/core';
+import { Grid, Typography } from '@material-ui/core';
+import {
+  EmailIcon,
+  EmailShareButton,
+  FacebookIcon,
+  FacebookShareButton,
+  TwitterIcon,
+  TwitterShareButton
+} from 'react-share';
 
 import DropDown from './DropDown';
 
 const useStyles = makeStyles(() => ({
   root: {},
-  source: {
+  social: {},
```

Comment:
@kilemensi Not sure if we are using this anywhere on this file?

Reply:
We use it here @nerdlet : https://github.com/CodeForAfrica/HURUmap-UI/pull/55/files#diff-265f38b306b25c5a58fb31a4a568f224R67